build/android/src/main/java/net.minetest.minetest/MtNativeActivity.java

```diff
@@ -19,6 +19,10 @@ public void onCreate(Bundle savedInstanceState) {
 	public void onDestroy() {
 		super.onDestroy();
 	}
+	
+	@Override
+	public void onBackPressed() {
```

Comment:
Try it with the changes I propose here: https://github.com/minetest/minetest/pull/4086#discussion_r63116305